src/NuGet.Core/NuGet.Configuration/Utility/ConfigurationContants.cs

```diff
@@ -25,8 +23,12 @@ public static class ConfigurationConstants
 
         public static readonly string Clear = "clear";
 
+        public static readonly string ClearTextPasswordAttribute = "clearTextPassword";
```

Comment:
Why do we need this and ClearTextPasswordToken?

Reply:
Because of ability to store certificates password in clear text without encryption

Reply:
Doesn't ClearTextPasswordToken contain the same content as this?

Reply:
I wrote independent tokens with similar naming to separate client cert functionality. If it is ok for you to reuse them - will delete this one :)

Reply:
If they are the same text, I think it's ok to reuse them.